eth/tracers/api.go

```diff
@@ -430,6 +434,14 @@ func (api *API) traceChain(ctx context.Context, start, end *types.Block, config
 	return sub, nil
 }
 
+func getBaseFee(isEspresso bool, sysCtx *core.SysContractCallCtx, tx *types.Transaction) *big.Int {
```

Comment:
Maybe it worth passing just the fee currency here, it woul make the function easier to test if we decide to do so.

Reply:
Done